src/pybind/mgr/dashboard/frontend/src/app/shared/components/language-selector/language-selector.component.ts

```diff
@@ -39,8 +39,16 @@ export class LanguageSelectorComponent implements OnInit {
     }
   }
 
+  /**
+   * Jest is being more restricted regarding spying on the reload method.
+   * This will allow us to spyOn this method instead.
+   */
+  windowReload() {

```

Comment:
Would it make sense to rename it to ``reloadWindow``? That seems to me to be more linguistically correct.

Reply:
Done